frontend/src/metabase/query_builder/reducers.js

```diff
@@ -500,16 +503,40 @@ export const currentState = handleActions(
   null,
 );
 
-export const timelineVisibility = handleActions(
+export const timelineIds = handleActions(
```

Comment:
nip: wonder if it'd be better to call it something like `visibleTImelineIds`? Just to make it clear this state handles timelines visibility on charts and nothing else

Reply:
Renamed this to `visibleTimelineIds` and `timelineEventIds` to `selectedTimelineIds`. Actions are renamed accordingly. Thanks!